### DISCUSSION TITLE
How to run `vercel dev` without next.js?

### DISCUSSION THREAD

**Discussion by cellis** [vercel/community#1960] [Help] [answered]
### Summary

I went through the [Serverless Quickstart](https://vercel.com/docs/concepts/functions/serverless-functions/quickstart) in order to deploy a server-only ( no website needed ) serverless function. The problem is `vercel dev` looks for `next` by default. How should we run dev on a typescript / javascript only project without the need for react/dom/next dependency? The docs don't specify a package json or an Express setup, and while it's not a huge deal to simply deploy a next.js project, I'd rather avoid building that as part of every deploy if I don't need it.

### Additional information

Note: I did read [this](https://vercel.com/guides/using-express-with-vercel), but I'm not trying to _deploy_ express. Rather I'd like to just test with express and deploy using `@vercel/node`.

> When we deploy a server (Express.js based application) within a [Serverless Function](https://vercel.com/docs/concepts/functions/serverless-functions), we are executing a full server implementation on each request. This is an anti-pattern, as Serverless Functions should serve one purpose.

**Comment by amyegan** [comment, marked as answer]
Hi @cellis! The `vercel dev` command will use the [Development Command](https://vercel.com/docs/concepts/deployments/configure-a-build#development-command) set for your project. You can change it in the project's [Build and Development Settings](https://vercel.com/docs/concepts/deployments/configure-a-build#build-and-development-settings). 

The guide you linked can help you set up a [Standalone Express](https://vercel.com/guides/using-express-with-vercel#standalone-express) app with Vercel. But if you only want to use Express for local testing, you should try changing the Development Command to control `vercel dev` behavior
